python/test/t_StandardSpaceCrossEntropyImportanceSampling_std.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+from openturns.testing import assert_almost_equal
+ot.RandomGenerator.SetSeed(1)
```

Comment:
OpenTURNS sets the seed to zero when starting. So when the unit test is executed from a brand new interpreter, there is not need to set the seed. Leaving the seed here may mean that the reference result is only correct for that particular seed, which is wrong here since the reference value is analytically computed.
```suggestion
```

Reply:
Ok, I switch the seed to zero.